templates/README.example.md

```diff
@@ -1,62 +1,48 @@
-[![Dependency Status](https://gemnasium.com/ADD_GEMNASIUM_TOKEN.svg)](https://gemnasium.com/renuo/<appname>) [![Build Status](https://travis-ci.com/renuo/<appname>.svg?token=ADD_TRAVIS_TOKEN&branch=master)](https://travis-ci.com/renuo/<appname>) [![Build Status](https://travis-ci.com/renuo/<appname>.svg?token=ADD_TRAVIS_TOKEN&branch=develop)](https://travis-ci.com/renuo/<appname>) [![Build Status](https://travis-ci.com/renuo/<appname>.svg?token=ADD_TRAVIS_TOKEN&branch=testing)](https://travis-ci.com/renuo/<appname>)
+# Project Title
+Short project description
 
-# <appname>
+## Environments
 
-## Important Links
+| Branch  | Domain                          | Deployment | CI                                      |
+| ------- | ------------------------------- | ---------- | --------------------------------------- |
+| develop | https://                        | auto       | [![Build Status][ci-develop-badge]][ci] |
+| testing | https://                        | auto       | [![Build Status][ci-testing-badge]][ci] |
+| master  | https://                        | release    | [![Build Status][ci-master-badge]][ci]  |
 
-* TODO: link to issue tracker, wiki, important resources, etc.
-* https://github.com/renuo/<appname>
-* https://<appname>-master.renuoapp.ch
-* https://<appname>-develop.renuoapp.ch
-* https://<appname>-testing.renuoapp.ch
+[ci]: https://travis-ci.com/renuo/<appname>
+[ci-develop-badge]: https://travis-ci.com/renuo/<url>
+[ci-testing-badge]: https://travis-ci.com/renuo/<url>
+[ci-master-badge]: https://travis-ci.com/renuo/<url>
```

Comment:
Also all the URL parameters are missing: `https://travis-ci.com/renuo/<appname>.svg?token=ADD_TRAVIS_TOKEN&branch=testing`